### DISCUSSION TITLE
METplus Coordinated 6.0.0 Release Acceptance Testing

### DISCUSSION THREAD

**Discussion by JohnHalleyGotway** [dtcenter/METplus#2342] [Release Acceptance Testing]
Listed below are summaries of the new features, enhancements, and bugfixes included in all development cycles for each software component of the **METplus Coordinated 6.0.0 release**. These tables summarize the status of external acceptance testing. This discussion is updated after each development release (e.g. beta1, beta2, beta3, rc1, and so on).

Please see the [METplus 6.0 Existing Builds](https://dtcenter.org/community-code/metplus/metplus-6-0-existing-builds) page to see where the release candidate versions are already installed.

<details>
<summary><h2>Instructions</h2></summary>

For each line in these tables, the `Tester` listed should test the *release candidate* to confirm the changes work as expected in their environment.

- For **Integration Testing** (not specifically tied to a single GitHub issue):
  - **ON THIS DISCUSSION**: Add a comment to document the testing outcome *using the template provided below*.
- For **Repository Testing** (new features, enhancements, and bugfixes):
  - **ON THE ISSUE ITSELF**: Document their testing with a comment *using the template provided below*.
  - **ON THIS DISCUSSION**: Add a comment linking to their issue comment created above.

Copy/paste the following template into your issue or discussion comment:

```
<h3>Release Acceptance Testing Summary</h3>

Version: METplus-6.0.0-beta1 or METplus-6.0.0-beta2
Date:
Location:
Status (PASS/FAIL):
Description:
```
The `Tester` is set to `NA` for any issues that do not require external acceptance testing.
</details>

<details>
<summary><h2>Integration Testing</h2></summary>

*Integration Testing* refers to testing functionality not specifically tied to a single GitHub issue. The largest category here is testing existing use cases in your environment (e.g. NOAA/EMC testing their EMC Vx System use cases). This also includes testing new features across multiple METplus components.

Users are encouraged to test their existing use cases with the METplus release candidate and report their results as comments on this discussion (see **Instructions** above). Please indicate the name of the use case if not already listed in the table below. Comments about expected results (**PASS**) and unexpected results (**FAIL**) are welcome. The METplus team will add a row for the use case and summarize those comments in the table below.

| Status<br>(OPEN,<br>PASS,<br>FAIL) | Description | Tester | Acceptance Testing Comment Link |
|---|---|---|---|
| | | | |

</details>

<details>
<summary><h2>Repository Testing: METplus Wrappers (<a href="https://github.com/orgs/dtcenter/projects/76">Project Board</a>)</h2></summary>

Testing of specific METplus Wrapper issues (new features, enhancements, and bugfixes).

Please see the METplus [Release Notes](https://metplus.readthedocs.io/en/develop/Users_Guide/release-notes.html#metplus-wrappers-release-notes) and [Upgrade Instructions](https://metplus.readthedocs.io/en/develop/Users_Guide/release-notes.html#metplus-wrappers-upgrade-instructions).

| Status<br>(OPEN,<br>PASS,<br>FAIL) | Topic | METplus Issue | Dev Cycle | Tester | Acceptance Testing Comment Link |
|---|---|---|---|---|---|
| PASS | | #1488, #2279, #2282, #2310, #2338  | beta1 | | No external testing required |
| PASS | | #2294 | beta1 | @LoganDawson-NOAA | PR [review comment](https://github.com/dtcenter/METplus/pull/2295#pullrequestreview-1569956815) |
| PASS | #2283 | beta1 | @j-opatz |  |
| **OPEN** | | #2299 | beta1 | @PerryShafran-NOAA, @Ho-ChunHuang-NOAA, @BinbinZhou-NOAA  | [METplus Wrappers Upgrade Instructions](https://metplus.readthedocs.io/en/develop/Users_Guide/release-notes.html#metplus-wrappers-upgrade-instructions) |
| PASS | Internal | #2253, #898, #2349,  #2405, #2123, #1667, #2329, #2420, #2377, #2412 | beta2 | | No external testing required |
| **OPEN** | WaveletStat | #2252 | beta2 | @mpm-meto | |
| PASS | SeriesAnalysis multiple input files | #2219 | beta2 | @j-opatz | |
| **OPEN** | TCDiag MET updates | #2340 | beta2 | @jvigh | |
| **OPEN** | Symbolic link to run_metplus.py | #2423 | beta2 | @DanielAdriaansen | |
| PASS | Internal | #2454, #2371 | beta3 | | No external testing required |
| **OPEN** | Use case: OMI | #1989 | beta3 | @malloryprow | |
| PASS | Use case: RRFS | #2406 | beta3 | @j-opatz | |
| **OPEN** | Use case: Satellite Altimetry | #2383 | beta3 | @bikegeek | |
| PASS | PointStat land mask | #2334 | beta3 | @j-opatz | |
| **OPEN** | TCPairs diag options | #2430 | beta3 | @JohnHalleyGotway | |
| **OPEN** | TCDiag new config options | #2432 | beta3 | @jvigh | |
| **OPEN** | Prevent error if inputs are missing | #2460 | beta3 | @ShannonShields-NOAA | |
| PASS | Internal | #2509, #2537, #2372, #2463 | beta4 | | No external testing required |
| PASS | ASCII2NC file window bad default and redundant init of wrappers | #2520 | beta4 | @malloryprow , @ShannonShields-NOAA | https://github.com/dtcenter/METplus/issues/2520#issuecomment-2383821632 |
| PASS | Internal | #2513 | beta5 | | No external testing required |
| **OPEN** | Wrapper: Madis2Nc  | #1514 | beta5 | @georgemccabe | See [I-WRF Use Case](https://github.com/NCAR/i-wrf/blob/main/use_cases/Hurricane_Matthew/METplus/PointStat_matthew.conf) |
| **OPEN** | Enhance TCI Use Case | #2388 | beta5 | @anewman89 | |
| **OPEN** | UGRID updates | #2433 | beta5 | @DanielAdriaansen | [PR Review](https://github.com/dtcenter/METplus/pull/2517#pullrequestreview-2022669641) |
| **OPEN** | MvMode config format changes | #2459 | beta5 | @hertneky | |
| **OPEN** | New new time_offset_warning option | #2479 | beta5 | @MarcelCaron-NOAA | |
| **OPEN** | ASCII2NC support valid beg end | #2547 | beta5 | Buoy data for NOAA/EMC or Land Project Use Case? | |
| **OPEN** | Use Case: Stratosphere QBO | #2557 | beta5 | @CPKalb | |
| PASS | Fix support for multiple PointStat obs files | #2567 | beta5 | @georgemccabe | See [I-WRF Use Case] |(https://github.com/NCAR/i-wrf/blob/main/use_cases/Hurricane_Matthew/METplus/PointStat_matthew.conf) |
| **OPEN** | Allow PCPCombine missing inputs | #2578 | beta5 | @mpm-meto | |
| **OPEN** | PBL use case reproducible results | #2246 | beta5 | @jason-english | |
| PASS | Internal | [#1882](https://github.com/dtcenter/METplus/issues/1882), [#2452](https://github.com/dtcenter/METplus/issues/2452), [#2489](https://github.com/dtcenter/METplus/issues/2489), [#2679](https://github.com/dtcenter/METplus/issues/2679), [#2695](https://github.com/dtcenter/METplus/issues/2695), [#2710](https://github.com/dtcenter/METplus/issues/2710), [#2714](https://github.com/dtcenter/METplus/issues/2714) | beta6 || No external testing required |
| **OPEN** | Bugfix: ASCII2NC file window bad default value | [#2520](https://github.com/dtcenter/METplus/issues/2520) | beta6 | @malloryprow ||
| **OPEN** | Point2Grid MET config | [#2540](https://github.com/dtcenter/METplus/issues/2540) | beta6 | @Ho-ChunHuang-NOAA ||
| PASS | Inconsistent RUN_ID | [#2596](https://github.com/dtcenter/METplus/issues/2596) | beta6 || No external testing required |
| PASS | Groups of forecast leads | [#2612](https://github.com/dtcenter/METplus/issues/2612) | beta6 | @CPKalb | [PR Review](https://github.com/dtcenter/METplus/pull/2698#pullrequestreview-2323224908) |
| **OPEN** | separate climo for fcst/obs | [#2622](https://github.com/dtcenter/METplus/issues/2622) | beta6 | @GwenChen-NOAA ||
| PASS | RRFS use case | [#2647](https://github.com/dtcenter/METplus/issues/2647) | beta6 ||No external testing required|
| **OPEN** | SeriesAnalysis -aggr | [#2651](https://github.com/dtcenter/METplus/issues/2651) | beta6 | @mpm-meto ||
| **OPEN** | SeriesAnalysis field info | [#2705](https://github.com/dtcenter/METplus/issues/2705) | beta6 | @wuxx66 ||
| **OPEN** | point_weight_flag | [#2727](https://github.com/dtcenter/METplus/issues/2727) | beta6 | @mpm-meto ||
| PASS | Internal | [#918](https://github.com/dtcenter/METplus/issues/918), [#1610](https://github.com/dtcenter/METplus/issues/1610), [#2300](https://github.com/dtcenter/METplus/issues/2300), [#2339](https://github.com/dtcenter/METplus/issues/2339), [#2560](https://github.com/dtcenter/METplus/issues/2560), [#2562](https://github.com/dtcenter/METplus/issues/2562), [#2597](https://github.com/dtcenter/METplus/issues/2597), [#2666](https://github.com/dtcenter/METplus/issues/2666), [#2682](https://github.com/dtcenter/METplus/issues/2682), [#2716](https://github.com/dtcenter/METplus/issues/2716), [#2737](https://github.com/dtcenter/METplus/issues/2737), [#2739](https://github.com/dtcenter/METplus/issues/2739), [#2751](https://github.com/dtcenter/METplus/issues/2751), [#2754](https://github.com/dtcenter/METplus/issues/2754), [#2757](https://github.com/dtcenter/METplus/issues/2757), [#2763](https://github.com/dtcenter/METplus/issues/2763), [#2772](https://github.com/dtcenter/METplus/issues/2772) | rc1 |||
| **OPEN** | 
Bugfix: StatAnalysis fcst_lev instead of fcst_level | [#2742](https://github.com/dtcenter/METplus/issues/2742) | rc1 | @briannen ||
| PASS | Bugfix: PCPCombine time in field info | [#2762](https://github.com/dtcenter/METplus/issues/2762) | rc1 | @CPKalb ||
</details>

<details>
<summary><h2>Repository Testing: MET (<a href="https://github.com/orgs/dtcenter/projects/74">Project Board</a>)</h2></summary>

Testing of specific MET issues (new features, enhancements, and bugfixes).

Please see the MET [Release Notes](https://met.readthedocs.io/en/develop/Users_Guide/release-notes.html#met-release-notes) and [Upgrade Instructions](https://met.readthedocs.io/en/develop/Users_Guide/release-notes.html#met-upgrade-instructions).

| Status<br>(OPEN,<br>PASS,<br>FAIL) | Topic | MET Issue | Dev Cycle | Tester | Acceptance Testing Comment Link |
|---|--|---|---|---|---|
| PASS | Internal | [METbaseimage#9](https://github.com/dtcenter/METbaseimage/issues/9), [METbaseimage#12](https://github.com/dtcenter/METbaseimage/issues/12), [#2643](https://github.com/dtcenter/MET/issues/2643), [#2650](https://github.com/dtcenter/MET/issues/2650), [#2659](https://github.com/dtcenter/MET/issues/2659), [#2665](https://github.com/dtcenter/MET/issues/2665), [#2669](https://github.com/dtcenter/MET/issues/2669), [#2682](https://github.com/dtcenter/MET/issues/2682) | beta1 | | No external testing required |
| PASS | TC-Diag | [#2609](https://github.com/dtcenter/MET/issues/2609) | beta1 | @jvigh | PR [review comment](https://github.com/dtcenter/MET/pull/2680#pullrequestreview-1623263536) |
| **OPEN** | CF NetCDF | [#2638](https://github.com/dtcenter/MET/issues/2638) | beta1 | @robdarvell | |
| **OPEN** | IODA2NC | [#2640](https://github.com/dtcenter/MET/issues/2640) | beta1 | @fearon10 | |
| **OPEN** | Percentile Thresholds | [#2644](https://github.com/dtcenter/MET/issues/2644) | beta1 | @malloryprow | |
| **OPEN** | MODE CTS | [#2648](https://github.com/dtcenter/MET/issues/2648) | beta1 | @jhicks14 | |
| PASS | Internal | [METbaseimage#13](https://github.com/dtcenter/METbaseimage/issues/13), [METbaseimage#15](https://github.com/dtcenter/METbaseimage/issues/15), [#2699](https://github.com/dtcenter/MET/issues/2699), [#2547](https://github.com/dtcenter/MET/issues/2547), [#2611](https://github.com/dtcenter/MET/issues/2611), [#2689](https://github.com/dtcenter/MET/issues/2689), [#2702](https://github.com/dtcenter/MET/issues/2702), [#2716](https://github.com/dtcenter/MET/issues/2716), [#2690](https://github.com/dtcenter/MET/issues/2690), [#2673](https://github.com/dtcenter/MET/issues/2673), [#2666](https://github.com/dtcenter/MET/issues/2666), [#2687](https://github.com/dtcenter/MET/issues/2687), [#2730](https://github.com/dtcenter/MET/issues/2730) | beta2 | | No external testing required |
| **OPEN** | Temporary Files | [#2691](https://github.com/dtcenter/MET/issues/2691) | beta2 | @johnlwagner | |
| **OPEN** | UGRID | [#2231](https://github.com/dtcenter/MET/issues/2231) | beta2 | @mpm-meto | |
| **OPEN** | TC-Pairs | [#2476](https://github.com/dtcenter/MET/issues/2476),[#2699](https://github.com/dtcenter/MET/issues/2699) | beta2 |  @musgrave-kate | |
| **OPEN** | TC-Diag | [#2550](https://github.com/dtcenter/MET/issues/2550) | beta2 | @musgrave-kate | |
| **OPEN** | MODE Convolution | [#2724](https://github.com/dtcenter/MET/issues/2724) | beta2 | ??? | |
| **OPEN** | MODE Multivar | [#2706](https://github.com/dtcenter/MET/issues/2706), [#2708](https://github.com/dtcenter/MET/issues/2708) | beta2 | ??? | |
| PASS | Internal | [METbaseimage#20](https://github.com/dtcenter/METbaseimage/issues/20), [#2757](https://github.com/dtcenter/MET/issues/2757), [#2760](https://github.com/dtcenter/MET/issues/2760), [#2776](https://github.com/dtcenter/MET/issues/2776), [#2785](https://github.com/dtcenter/MET/issues/2785), [#2792](https://github.com/dtcenter/MET/issues/2792), [#2796](https://github.com/dtcenter/MET/issues/2796) | beta3 | | No external testing required |
| **OPEN** | Fix NSIDC NetCDF | [#2652](https://github.com/dtcenter/MET/issues/2652) | beta3 | ??? | |
| **OPEN** | Fix Py Embed Point Data | [#2755](https://github.com/dtcenter/MET/issues/2755) | beta3 | ??? | |
| **OPEN** | Fix GRIB2 4.48 template | [#2782](https://github.com/dtcenter/MET/issues/2782) | beta3 | ??? | |
| **OPEN** | Native WRF | [#2547](https://github.com/dtcenter/MET/issues/2547) | beta3 | @jaredalee | |
| **OPEN** | ISMN Point Obs | [#2701](https://github.com/dtcenter/MET/issues/2701) | beta3 | @anewman89 | |
| **OPEN** | MODE Multivar | [#2745](https://github.com/dtcenter/MET/issues/2745) | beta3 | ??? | |
| **OPEN** | TC-Diag | [#2769](https://github.com/dtcenter/MET/issues/2769) | beta3 | @jvigh @KathrynNewman | |
| **OPEN** | Python Temp Files | [#2772](https://github.com/dtcenter/MET/issues/2772) | beta3 | @DanielAdriaansen | |
| **OPEN** | Warning Messages | [#2801](https://github.com/dtcenter/MET/issues/2801) | beta3 | Marcel Caron | |
| **OPEN** | Stat-Analysis `-set_hdr` | [#2805](https://github.com/dtcenter/MET/issues/2805) | beta3 | @jvigh | |
| **OPEN** | LAEA MET NetCDF | [#2809](https://github.com/dtcenter/MET/issues/2809) | beta3 | @mpm-meto | |
| PASS | Internal  | [#2379](https://github.com/dtcenter/MET/issues/2379), [#2696](https://github.com/dtcenter/MET/issues/2696), [#2815](https://github.com/dtcenter/MET/issues/2815), [#2673](https://github.com/dtcenter/MET/issues/2673), [#2830](https://github.com/dtcenter/MET/issues/2830) | beta4 | | |
| **OPEN** | Fix range/azimuth grids | [#2833](https://github.com/dtcenter/MET/issues/2833) | beta4 | | |
| **OPEN** | Ensemble probabilities | [#2280](https://github.com/dtcenter/MET/issues/2280) | beta4 | | |
| PASS | Wind direction stats | [#2395](https://github.com/dtcenter/MET/issues/2395) | beta4 and beta5 | @malloryprow | https://github.com/dtcenter/MET/issues/2395#issuecomment-2383842764 |
| **OPEN** | ECNT obs error stats | [#2583](https://github.com/dtcenter/MET/issues/2583) | beta4 | | |
| **OPEN** | Arctic buoy obs | [#2654](https://github.com/dtcenter/MET/issues/2654) | beta4 | | |
| **OPEN** | MvMODE efficiency | [#2707](https://github.com/dtcenter/MET/issues/2707) | beta4 | | |
| **OPEN** | NetCDF point obs to Pandas DataFrame | [#2781](https://github.com/dtcenter/MET/issues/2781) | beta5 | | |
| **OPEN** | RPSS from prob | [#2786](https://github.com/dtcenter/MET/issues/2786) | beta4 | | |
| **OPEN** | MODE convex hull | [#2819](https://github.com/dtcenter/MET/issues/2819) | beta4 | | |
| **OPEN** | Grid-Stat conf SEEPS climo  | [#2601](https://github.com/dtcenter/MET/issues/2601) | beta5 | @RachelNorth | |
| **OPEN** | Fix wind computations | [#2841](https://github.com/dtcenter/MET/issues/2841) | beta5 | | |
| **OPEN** | New -ugrid_config option | [#2842](https://github.com/dtcenter/MET/issues/2842) | beta5 | | |
| **OPEN** | Modified AOD QC | [#2853](https://github.com/dtcenter/MET/issues/2853) | beta5 | @Ho-ChunHuang-NOAA | |
| **OPEN** | Fix ens stat climo handling | [#2856](https://github.com/dtcenter/MET/issues/2856) | beta5 | | |
| **OPEN** | Point2Grid tripolar datasets | [#2857](https://github.com/dtcenter/MET/issues/2857) | beta5 | | |
| **OPEN** | NOAA Unstructured grids | [#2860](https://github.com/dtcenter/MET/issues/2860) | beta5 | @SamiraArdani-NOAA  | |
| **OPEN** | Point2Grid -qc for ADP | [#2867](https://github.com/dtcenter/MET/issues/2867) | beta5 | @Ho-ChunHuang-NOAA | |
| **OPEN** | Omit warning for MISSING | [#2870](https://github.com/dtcenter/MET/issues/2870) | beta5 | | |
| **OPEN** | Allow PCP-Combine missing data | [#2883](https://github.com/dtcenter/MET/issues/2883) | beta5 | | |
| **OPEN** | Fix Stat-Analysis options | [#2888](https://github.com/dtcenter/MET/issues/2888) | beta5 | | |
| **OPEN** | Fix handling of point obs | [#2897](https://github.com/dtcenter/MET/issues/2897) | beta5 | | |
| **OPEN** | TC-Stat for -set_hdr | [#2911](https://github.com/dtcenter/MET/issues/2911) | beta5 | @mollybsmith-noaa | |
| **OPEN** | Refine PB2NC error messages | [#2912](https://github.com/dtcenter/MET/issues/2912) | beta5 | @malloryprow | |
| PASS | Level mismatch warning | [#2795](https://github.com/dtcenter/MET/issues/2795) | beta5 | @malloryprow and @MarcelCaron-NOAA | https://github.com/dtcenter/MET/issues/2795#issuecomment-2383855364 |
| PASS | Internal | [METbaseimage#27](https://github.com/dtcenter/METbaseimage/issues/27), [METplus-Internal#23](https://github.com/dtcenter/METplus-Internal/issues/23), [#2023](https://github.com/dtcenter/MET/issues/2023), [#2673](https://github.com/dtcenter/MET/issues/2673), [#2936](https://github.com/dtcenter/MET/issues/2936), [#2937](https://github.com/dtcenter/MET/issues/2937), [#2948](https://github.com/dtcenter/MET/issues/2948), [#2949](https://github.com/dtcenter/MET/issues/2949), [#2968](https://github.com/dtcenter/MET/issues/2968) | beta6 | | No external testing required |
| **OPEN** | Fix CTS statistic bugs | [#2958](https://github.com/dtcenter/MET/issues/2958) | beta6 | @jprestop | See [PR Review](https://github.com/dtcenter/MET/pull/2959#pullrequestreview-2283387205) |
| **OPEN** | MTD grid projetions | [#2979](https://github.com/dtcenter/MET/issues/2979) | beta6 | @anwe5 ||
| **OPEN** | Series-Analysis increments | [#1371](https://github.com/dtcenter/MET/issues/1371) | beta6 | @mpm-meto ||
| **OPEN** | set_attr_grid | [#1729](https://github.com/dtcenter/MET/issues/1729) | beta6 | @michelleharrold ||
| **OPEN** | point_weight_flag | [#2279](https://github.com/dtcenter/MET/issues/2279) | beta6 | @mpm-meto ||
| **OPEN** | Observation anomaly ACC | [#2308](https://github.com/dtcenter/MET/issues/2308) | beta6 | @GwenChen-NOAA ||
| **OPEN** | Point2Grid QC | [#2880](https://github.com/dtcenter/MET/issues/2880) | beta6 | @Ho-ChunHuang-NOAA ||
| **OPEN** | SEEPS updates | [#2882](https://github.com/dtcenter/MET/issues/2882) | beta6 | @mpm-meto ||
| PASS | Weighted contingency tables | [#2887](https://github.com/dtcenter/MET/issues/2887) | beta6 | @j-opatz | See [PR Review](https://github.com/dtcenter/MET/pull/2988#pullrequestreview-2361555318) |
| **OPEN** | MPR OBTYPE column | [#2893](https://github.com/dtcenter/MET/issues/2893) | beta6 | @robdarvell ||
| **OPEN** | Forecast climatology | [#2924](https://github.com/dtcenter/MET/issues/2924) | beta6 | @GwenChen-NOAA ||
| **OPEN** | PB2NC Bufr warnings | [#2938](https://github.com/dtcenter/MET/issues/2938) | beta6 | @malloryprow ||

</details>

<details>
<summary><h2>Repository Testing: METplus Analysis (<a href="https://github.com/orgs/dtcenter/projects/75">Project Board</a>)</h2></summary>

Testing of specific METplus Analysis (METdataio, METcalcpy, METplotpy, METviewer, and METexpress) issues (new features, enhancements, and bugfixes).

Please see the METdataio [Release Notes](https://metdataio.readthedocs.io/en/develop/Users_Guide/release-notes.html#metdataio-release-notes) and [Upgrade Instructions](https://metdataio.readthedocs.io/en/develop/Users_Guide/release-notes.html#metdataio-upgrade-instructions).

| Status<br>(OPEN,<br>PASS,<br>FAIL)  | Topic | METdataio Issue | Dev Cycle | Tester | Acceptance Testing Comment Link |
|---|---|---|---|---|---|
|PASS| | [#219](https://github.com/dtcenter/METdataio/issues/219), [#221](https://github.com/dtcenter/METdataio/issues/221) |PASS| |[#228](https://github.com/dtcenter/METdataio/issues/228) |beta1||No External Testing Required|
|PASS| | [#238](https://github.com/dtcenter/METdataio/issues/238) |beta2||No External Testing Required|
|PASS| | [#232](https://github.com/dtcenter/METdataio/issues/232), [#247](https://github.com/dtcenter/METdataio/issues/247), [#256](https://github.com/dtcenter/METdataio/issues/256), [#264](https://github.com/dtcenter/METdataio/issues/264), [#267](https://github.com/dtcenter/METdataio/issues/267)|beta3||No External Testing Required|
|PASS| | [#234](https://github.com/dtcenter/METdataio/issues/234) |beta3|@VanderleiVargas-NOAA||
|PASS| | [#245](https://github.com/dtcenter/METdataio/issues/245) |beta3|@mollybsmith-noaa||
|PASS| | [#240](https://github.com/dtcenter/METdataio/issues/240), [#273](https://github.com/dtcenter/METdataio/issues/273), [#282](https://github.com/dtcenter/METdataio/issues/282), [#289](https://github.com/dtcenter/METdataio/issues/289) |beta4||No External Testing Required
|PASS| | [#231](https://github.com/dtcenter/METdataio/issues/231), [#255](https://github.com/dtcenter/METdataio/issues/255), [#272](https://github.com/dtcenter/METdataio/issues/272), [#307](https://github.com/dtcenter/METdataio/issues/307), [METplus-Internal #53](https://github.com/dtcenter/METplus-Internal/issues/53) |beta5||No External Testing Required|
|PASS| | [#35](https://github.com/dtcenter/METdataio/issues/35), [METplus-Internal #45](https://github.com/dtcenter/METplus-Internal/issues/45), [METplus-Internal #50](https://github.com/dtcenter/METplus-Internal/issues/50), [METplus-Internal #56](https://github.com/dtcenter/METplus-Internal/issues/56), [#202](https://github.com/dtcenter/METdataio/issues/202), [#226](https://github.com/dtcenter/METdataio/issues/226), [#320](https://github.com/dtcenter/METdataio/issues/320), [#321](https://github.com/dtcenter/METdataio/issues/321), [#335](https://github.com/dtcenter/METdataio/issues/335), [#272](https://github.com/dtcenter/METdataio/issues/272), [#318](https://github.com/dtcenter/METdataio/issues/318), [#327](https://github.com/dtcenter/METdataio/pull/327), [PR#341](https://github.com/dtcenter/METdataio/pull/341) |beta6||No External Testing Required|
|PASS| | [#337](https://github.com/dtcenter/METdataio/issues/337)|rc1||No External Testing Required|
| | | | | | |

Please see the METcalcpy [Release Notes](https://metcalpy.readthedocs.io/en/develop/Users_Guide/release-notes.html#metcalcpy-release-notes) and [Upgrade Instructions](https://metcalpy.readthedocs.io/en/develop/Users_Guide/release-notes.html#metcalpy-upgrade-instructions).

| Status<br>(OPEN,<br>PASS,<br>FAIL) | Topic | METcalcpy Issue | Dev Cycle | Tester | Acceptance Testing Comment Link |
|---|---|---|---|---|---|
|PASS| | [#322](https://github.com/dtcenter/METcalcpy/issues/322) |beta1||No External Testing Required|
|PASS| | [#345](https://github.com/dtcenter/METcalcpy/issues/345) |beta3||No External Testing Required|
|PASS| | [#325](https://github.com/dtcenter/METcalcpy/issues/325) |beta3|@VanderleiVargas-NOAA  ||
|PASS| | [#329](https://github.com/dtcenter/METcalcpy/issues/329) |beta3|@mollybsmith-noaa ||
|PASS| | [#326](https://github.com/dtcenter/METcalcpy/issues/326), [#349](https://github.com/dtcenter/METcalcpy/issues/349), [#384](https://github.com/dtcenter/METcalcpy/issues/384), [#385](https://github.com/dtcenter/METcalcpy/issues/385), [METplus-Internal #54](https://github.com/dtcenter/METplus-Internal/issues/54) |beta5||No External Testing Required|
|PASS| | [METplus-Internal #46](https://github.com/dtcenter/METplus-Internal/issues/46), [#360](https://github.com/dtcenter/METcalcpy/issues/360)|beta6||No External Testing Required|
|PASS| | |rc1|no RC1 issues|No External Testing Required|
| | | | | | |

Please see the METplotpy [Release Notes](https://metplotpy.readthedocs.io/en/develop/Users_Guide/release-notes.html#metplotpy-release-notes) and [Upgrade Instructions](https://metplotpy.readthedocs.io/en/develop/Users_Guide/release-notes.html#metplotpy-upgrade-instructions).

| Status<br>(OPEN,<br>PASS,<br>FAIL) | Topic | METplotpy Issue | Dev Cycle | Tester | Acceptance Testing Comment Link |
|---|---|---|---|---|---|
|PASS| | [#385](https://github.com/dtcenter/METplotpy/issues/385) |beta1||No External Testing Required|
|PASS| | [#345](https://github.com/dtcenter/METplotpy/issues/345) |beta3||No External Testing Required|
|PASS| | [#414](https://github.com/dtcenter/METplotpy/issues/414), [#425](https://github.com/dtcenter/METplotpy/issues/425), [#429](https://github.com/dtcenter/METplotpy/issues/429) |beta4||No External Testing Required|
|PASS| | [#342](https://github.com/dtcenter/METplotpy/issues/342), [#380](https://github.com/dtcenter/METplotpy/issues/380), [#383](https://github.com/dtcenter/METplotpy/issues/383), [#388](https://github.com/dtcenter/METplotpy/issues/388), [#417](https://github.com/dtcenter/METplotpy/issues/417), [METplus-Internal #55](https://github.com/dtcenter/METplus-Internal/issues/55) |beta5||No External Testing Required| 
|PASS| | [METplus-Internal #57](https://github.com/dtcenter/METplus-internal/issues/57), [#233](https://github.com/dtcenter/METplotpy/issues/233), [#461](https://github.com/dtcenter/METplotpy/issues/461) |beta6||No External Testing Required| 
|PASS| |  |rc1| no RC1 issues|No External Testing Required| 
| | | | | | |

Please see the METviewer [Release Notes](https://metviewer.readthedocs.io/en/develop/Users_Guide/release-notes.html#metviewer-release-notes) and [Upgrade Instructions](https://metviewer.readthedocs.io/en/develop/Users_Guide/release-notes.html#metviewer-upgrade-instructions).

| Status<br>(OPEN,<br>PASS,<br>FAIL) | Topic | METviewer Issue | Dev Cycle | Tester | Acceptance Testing Comment Link |
|---|---|---|---|---|---|
|PASS|None|beta1| |No beta1 issues|
|PASS|| [#387](https://github.com/dtcenter/METviewer/issues/387), [#465](https://github.com/dtcenter/METviewer/issues/465), [#494](https://github.com/dtcenter/METviewer/issues/494), [#495](https://github.com/dtcenter/METviewer/issues/495), [#497](https://github.com/dtcenter/METviewer/issues/497), [#501](https://github.com/dtcenter/METviewer/issues/501) |beta2||No External Testing Required|
|PASS|| [#506](https://github.com/dtcenter/METviewer/issues/506), [#508](https://github.com/dtcenter/METviewer/issues/508) |beta3||No External Testing Required|
|PASS|| [#510](https://github.com/dtcenter/METviewer/issues/510), [#513](https://github.com/dtcenter/METviewer/issues/513), [#516](https://github.com/dtcenter/METviewer/issues/516) |beta4||No External Testing Required|
|PASS|| [#491](https://github.com/dtcenter/METviewer/issues/491), [#521](https://github.com/dtcenter/METviewer/issues/521), 
|PASS||[METplus-Internal #58](https://github.com/dtcenter/METplus-Internal/issues/58) |beta5||No External Testing Required|
|PASS||[METplus-Internal #29](https://github.com/dtcenter/METplus-Internal/issues/29) |beta6||No External Testing Required|
|PASS||[METplus-Internal #60](https://github.com/dtcenter/METplus-Internal/issues/60) |beta6||No External Testing Required|
|PASS||[#517](https://github.com/dtcenter/METviewerl/issues/517) |beta6||No External Testing Required|
|PASS|||rc1|no RC1 issues|No External Testing Required|
| | | | | | |

</details>

**Comment by Ho-ChunHuang-NOAA** [comment]
Hi, John:

I need permission to access the link to " METplus 6.0 Existing Builds".
Or simply tell me the modules load statements on the WCOSS2.

I assume you want me to test the ascii2nc utility for EVS AQM verification
still working in v6 beta1 version for #2299.

Ho-Chun


--

Ho-Chun Huang, Ph.D.

Physical Scientist III, Contractor with Lynker in Support of

NOAA/NWS/NCEP/EMC, U.S. Department of Commerce

5830 University Research Ct., Rm. 2792

College Park, MD 20740

***@***.*** ***@***.***>

301-683-3958

**Reply by jprestop** [reply]
Hi @Ho-ChunHuang-NOAA.   Apologies for the link for the [METplus 6.0 Existing Builds](https://dtcenter.org/community-code/metplus/metplus-6-0-existing-builds) page not initially working.  It should work for you now.  It was previously in "Draft" mode because there was no content.  At this time, there are only placeholders, as there are no full installations of the new beta release yet, except for on WCOSS2 Acorn.  However, testing of the actual code by users cannot be done on Acorn.  So testing of #2299, would need to take place on WCOSS2 Dogwood/Cactus or on Hera.  It is my understanding that the installation of the beta1 releases has not been requested for installation on WCOSS2 Dogwood/Cactus yet. I'd advise you to talk with @AliciaBentley-NOAA about the timeline for that.  There have been some unexpected installation issues on Hera, but we hope to have those resolved by early next week, at which time, I will post the appropriate modules to load on the Existing Builds page.  Please let us know if you have any questions.

**Comment by Ho-ChunHuang-NOAA** [comment]
Hi, Julie:

Thanks for the update.  I prefer to test the EVSv1.0 AQM code on the
Cactus/Dogwood.    I will wait until it is installed on the Cactus/Dogwood.

Ho-Chun

--

Ho-Chun Huang, Ph.D.

Physical Scientist III, Contractor with Lynker in Support of

NOAA/NWS/NCEP/EMC, U.S. Department of Commerce

5830 University Research Ct., Rm. 2792

College Park, MD 20740

***@***.*** ***@***.***>

301-683-3958

**Reply by jprestop** [reply]
@Ho-ChunHuang-NOAA Just a heads up that the beta1 release will only be installed using EVS v2.0.  I believe that shouldn't really impact you much other than having to load EVS v2.0 instead of EVS v1.0.

**Comment by Ho-ChunHuang-NOAA** [comment]
Hi, Julie:

I do not expect there will be a change of using ASCII2NC function for the
AQM verification between EVSv1.0 and EVSv2.0.  It is easier for me to test
it using EVSv1.0 then I can move on to other tasks.

Thanks for the head-up.

Ho-Chun

--

Ho-Chun Huang, Ph.D.

Physical Scientist III, Contractor with Lynker in Support of

NOAA/NWS/NCEP/EMC, U.S. Department of Commerce

5830 University Research Ct., Rm. 2792

College Park, MD 20740

***@***.*** ***@***.***>

301-683-3958

**Comment by malloryprow** [comment]
https://github.com/dtcenter/METplus/issues/2520#issuecomment-2383821632

**Comment by malloryprow** [comment]
https://github.com/dtcenter/MET/issues/2395#issuecomment-2383842764

**Comment by malloryprow** [comment]
https://github.com/dtcenter/MET/issues/2795

**Comment by ShannonShields-NOAA** [comment]
Release Acceptance Testing Summary
Version: MET-12.0.0-beta5
Date: 9/30/2024
Location: WCOSS2
Status (PASS/FAIL): PASS
Description: EVS subseasonal component was tested with beta5 and all prep, stats, and plots jobs were confirmed successful.
